### PR TITLE
Disable some ESLint warnings

### PR DESCRIPTION
**Pull request by MikeTaylor**
I want a clean slate before I start work on UIIN-2197.

### CODE REVIEW

**Review by zburke — COMMENTED** [review]
I _really_ appreciate the comments you added explaining why you're disabling lint where you do! But, I'm not a maintainer on this project any longer so disabling things isn't my call :) 

@mkuklis?

File: src/hooks/useInventoryBrowse/useInventoryBrowse.js [hunk 1]
```diff
@@ -86,6 +86,7 @@ const useInventoryBrowse = ({
 
   const updatePage = useCallback((_askAmount, _index, _firstIndex, direction) => {
     // TODO: implement prev-next pagination
+    // eslint-disable-next-line no-console
     console.log('direction', direction);
```
Comment (mkuklis) [inline src/hooks/useInventoryBrowse/useInventoryBrowse.js:90]: @MikeTaylor let's just remove this `console.log`

**Comment by MikeTaylor** [comment]
I am wary of removing actual code that someone added, presumably intentionally. So I'm happier removing the ESLint warning instead (especially given that there is an easy-to-grep `TODO` comment in place).

**Comment by MikeTaylor** [comment]
On further overnight reflection, you're right. I removed the `console.log` because the TODO comment makes the relevant place in the code easy to find without it.

**Pull request merged**